Sources/AppStoreConnectCLI/Model/BuildLocalization.swift

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import Model
+import SwiftyTextTable
+
+extension Model.BuildLocalization {
+    init(_ localization: AppStoreConnect_Swift_SDK.BetaBuildLocalization) {
+        self.init(
+            locale: localization.attributes?.locale,
+            whatsNew: localization.attributes?.whatsNew
+        )
+    }
+}
+
+extension Model.BuildLocalization: ResultRenderable, TableInfoProvider {
+
+    static func tableColumns() -> [TextTableColumn] {
+        [
+            TextTableColumn(header: "Locale"),
+            TextTableColumn(header: "What's New"),
+        ]
+    }
+
+    var tableRow: [CustomStringConvertible] {
+        [
+            locale ?? "",
+            "Render whatsNew info in table is currently not support, please use --json or --yaml instead",
+        ]
```

Comment:
Might just be able to `fatalError("unimplemented")` here. Since we are trying to ensure it's not being called